crates/bevy_pbr/src/environment_map/mod.rs

```diff
@@ -51,8 +51,7 @@ pub struct EnvironmentMapLight {
     pub diffuse_map: Handle<Image>,
     pub specular_map: Handle<Image>,
     /// TODO: Docs
-    // TODO: What to call this?
-    pub brightness: f32,
+    pub intensity: f32,
```

Comment:
Perhaps: "Scale factor for `diffuse_map` and `specular_map`. After multiplying by this value, all samples should be in units of cd/m^2"